### PR TITLE
The dog feature

### PR DESCRIPTION
**Pull request by bfudge**
This is the new dog feature to pull up dog gifs.

Refer to product requirements for more info.

### CODE REVIEW

**Comment by SLandgra** [comment]
Button still says cat, change that to dog.

**Comment by bfudge** [comment]
Ok, thanks dude.

File: cat-button-solved.html [hunk 1]
```diff
@@ -16,14 +16,14 @@
     $("#cat-button").on("click", function() {
 
       // Storing our giphy API URL for a random cat image
-      var queryURL = "https://api.giphy.com/v1/gifs/random?api_key=dc6zaTOxFJmzC&tag=cats";
+      var queryURL = "https://api.giphy.com/v1/gifs/random?api_key=dc6zaTOxFJmzC&tag=dogs";
```
Comment (bfudge) [inline cat-button-solved.html]: look around for other cat references, particularly:

Comment (bfudge) [reply cat-button-solved.html]: https://github.com/bfudge/cat-example/pull/1/files#diff-ea2ca05716d52da1f858523784d37345L10

**Pull request merged**